analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -101,7 +102,7 @@ abstract class AbstractIntegrationSpec : StringSpec() {
             val managedFiles = PackageManager.findManagedFiles(downloadResult.downloadDirectory)
 
             managedFiles.size shouldBe expectedManagedFiles.size
-            managedFiles.forEach { (manager, files) ->
+            managedFiles.entries.forAll { (manager, files) ->
```

Comment:
I'm curious: Why does `entries` need to be added here?

Reply:
Because `forAll` is not available for the type `Map`.